pkgs/applications/networking/instant-messengers/twinkle/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  libxml2,
+  libsndfile,
+  file,
+  readline,
+  bison,
+  flex,
+  ucommon,
+  ccrtp,
+  qt5,
+  alsaLib,
+  speex,
+  ilbc,
+  wrapQtAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "twinkle";
+  version = "1.10.2";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "LubosD";
+    rev = "v${version}";
+    sha256 = "0s0gi03xwvzp02ah4q6j33r9jx9nbayr6dxlg2ck9pwbay1nq1hx";
+  };
+
+  buildInputs = [
+    libxml2
+    file # libmagic
+    libsndfile
+    readline
+    ucommon
+    ccrtp
+    qt5.qtbase
+    qt5.qttools
+    qt5.qtquickcontrols2
+    alsaLib
+    speex
+    ilbc
+  ];
+
+  nativeBuildInputs = [ cmake bison flex wrapQtAppsHook ];
+
+  cmakeFlags = [ /* "-DWITH_G729=On" bcg729 missing */ "-DWITH_SPEEX=On" "-DWITH_ILBC=On" /* "-DWITH_DIAMONDCARD=On" breaks compilation */ ];
```

Comment:
`bcg729` was added to nixpkgs in #74043 :)

Reply:
Still won't compile, created upstream issue https://github.com/LubosD/twinkle/issues/176